modules/claims_api/app/sidekiq/claims_api/poa_updater.rb

```diff
@@ -28,7 +28,9 @@ def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
 
         ClaimsApi::Logger.log('poa', poa_id: poa_form.id, detail: 'BIRLS Success')
 
-        ClaimsApi::PoaVBMSUpdater.perform_async(poa_form.id) if enable_vbms_access?(poa_form:)
+        if enable_vbms_access?(poa_form:) && !Flipper.enabled?(:lighthouse_claims_api_poa_use_bd)
```

Comment:
this flipper check shouldn't be added here - we'll want the vbms updater to run either way, we just want to switch the ability to submit to VBMS <> Benefits Documents, so lines 27-32 in the poa_form_builder_job

this job still needs to run on every request if enable_vbms_access?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/17587/commits/e56deb8c19f7b0094c7f189c4200e77d1b2eb966